tee-worker/litentry/core/assertion-build/src/a11.rs

```diff
@@ -60,30 +60,31 @@ pub fn build(
 
 		if let Identity::Evm { network, address } = id {
 			if matches!(network, EvmNetwork::Ethereum) {
-				let address = from_utf8(address.as_ref()).unwrap().to_string();
-				let addresses = vec![address];
+				if let Ok(addr) = from_utf8(address.as_ref()) {
```

Comment:
Do we need to pass it to parentchain instead of ignoring it? @jingleizhang 

I see something in https://github.com/litentry/litentry-parachain/blob/656c1985e6c682f34ae5d4a7dbc1ad0653d67acd/primitives/core/src/error.rs#L64-L65

Is this supposed to be used?

Reply:
Agree. I think we should propagate it out.

Reply:
In those contexts(A4/A7/A10/A11) no need to propagate `ParseError` to parachain:
1. those payload are integer(Balance), return `ParseError` will be confusing.
2. in `from_utf8(address.as_ref())`, `address` is from `IDGraph`
3. we can add logs, if parse error.

Reply:
agree, some of ParseErrors are totally internal, and don't need to be propagated out